src/styles/blocks/phone.scss

```diff
@@ -0,0 +1,31 @@
+@use "../utils/mixins" as *;
+
+.phone {
+  &__label {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 12px;
+    line-height: 16px;
+
+    letter-spacing: 2px;
+    text-transform: uppercase;
+
+    color: $light-black;
+    position: absolute;
+    transform: translateX(calc(-100% - 20px)) translateY(5px);
+  }
+
+  &__icon {
+    width: 18px;
+    height: 18px;
+
+    @include for-tablet-and-desktop() {
+      width: $icons-width;
+      height: $icons-height;
+    }
+  }
+
+  &:not(:hover) &__label {
```

Comment:
generally it is an anti pattern __label should be always hidden and on hover you should show it :) just reverse the style rules

Reply:
bump my suggestion again :)